x/blob/README.md

```diff
@@ -2,49 +2,192 @@
 
 ## Abstract
 
-The `x/blob` module enables users to pay for arbitrary data to be published to the Celestia blockchain. Users create a single `BlobTx` that is composed of:
+The `x/blob` module enables users to pay for arbitrary data to be published to
+the Celestia blockchain. Users create a single `BlobTx` that is composed of:
 
-1. Multiple `Blob`s (Binary Large OBjects): the data they wish to publish. A single `Blob` is composed of:
+1. Multiple `Blob`s (Binary Large OBjects): the data they wish to publish. A
+   single `Blob` is composed of:
     1. `NamespaceId  []byte`: the namespace this blob should be published to.
     1. `Data         []byte`: the data to be published.
-    1. `ShareVersion uint32`: the version of the share format used to encode this blob into a share.
-1. A single `sdk.Tx` which is composed of:
+    1. `ShareVersion uint32`: the version of the share format used to encode
+       this blob into a share.
+1. A single [`sdk.Tx`](https://github.com/celestiaorg/cosmos-sdk/blob/v1.15.0-sdk-v0.46.13/docs/architecture/adr-020-protobuf-transaction-encoding.md) which is composed of:
```

Comment:
[nit] 
```suggestion
1. A single [`sdk.Tx`](https://github.com/celestiaorg/cosmos-sdk/blob/v1.15.0-sdk-v0.46.13/docs/architecture/adr-020-protobuf-transaction-encoding.md) which encapsulates a `MsgPayFroBlob` message that is composed of:
```

Reply:
[nit^2]

typo in `MsgPayFroBlob` also I think there's only a plural `MsgPayForBlobs`

Reply:
Thanks, fixed it :)